platform-service/sample.env

```diff
@@ -31,6 +31,7 @@ FLIPT_SERVICE_AVAILABLE=False
 # Cost calculation related ENVs
 MODEL_PRICES_URL="https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json"
 MODEL_PRICES_TTL_IN_DAYS=7
-MODEL_PRICES_FILE_PATH="/tmp/model_prices.json"
+MODEL_PRICES_FILE_PATH="fsspec-test/cost/model_prices.json"
+FILE_STORAGE='{"provider": "gcs", "credentials": {"token": "/path/to/google/creds.json"}}'
```

Comment:
@gaya3-zipstack Please rename to `FILE_STORAGE_CREDENTIALS`.

Reply:
Done